package.json

```diff
@@ -24,6 +24,7 @@
   },
   "scripts": {
     "start": "npm-run-all lint build run",
+    "package": "npm run build && cp web-ext-artifacts/`ls -t1 web-ext-artifacts | head -n 1` addon.xpi",
```

Comment:
Would it just be easier to delete all the old add-ons in ./web-ext-artifacts/*.zip before running `web-ext build`?
That way you could just do something like `cp web-ext-artifacts/*.zip addon.xpi` since we know ./web-ext-artifacts/ will only ever have at most 1 add-on ZIP file.

Reply:
@pdehaan Sure but I kind of like the idea of having a history of all the versions, seems convenient. What do you think?

Reply:
We keep a copy of all the released versions on t.f.c, for the record